lib/uzlib/tinflate.c

```diff
@@ -35,6 +35,12 @@
 #include <assert.h>
 #include "tinf.h"
 
+#if defined(__GNUC__) && (__GNUC__ >= 11)
```

Comment:
Why `__GNUC__ >= 11`?

It seems that optimize was introduced in GCC 4.4.0: https://gcc.gnu.org/onlinedocs/gcc-4.4.0/gcc/Function-Attributes.html#index-g_t_0040code_007boptimize_007d-function-attribute-2243